VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -3384,6 +3384,190 @@ public void Get_EmptyLiteralRaceFields()
+                { "CITYCODE_D", "00000" },
+                { "LONG_D", "-77.050636" },
+                { "LAT_D", "38.889248" },
+                { "SPOUSELV", "1" },
+                { "SPOUSEL", "Gazette" },
+                { "STNUM_R", "1829" },
+                { "PREDIR_R", "North" },
+                { "STNAME_R", "Charles" },
+                { "STDESIG_R", "Avenue" },
+                { "POSTDIR_R", "Southeast" },
+                { "UNITNUM_R", "Apt 2B" },
+                { "CITYTEXT_R", "Hartford" },
+                { "ZIP9_R", "06107" },
+                { "COUNTYTEXT_R", "Hartford" },
+                { "STATETEXT_R", "Connecticut" },
+                { "COUNTRYTEXT_R", "United States" },
+                { "ADDRESS_R", "4437 North Charles Avenue Southeast Apt 2B" },
+                { "DETHNICE", "233" },
+                { "DDADF", "John" },
+                { "DMOMF", "Momfirst" },
+                { "DMOMMDN", "Suzette" },
+                { "REFERRED", "Y" },
+                { "POILITRL", "Home" },
+                { "HOWINJ", "drug toxicity" },
+                { "TRANSPRT", "PE" },
+                { "COUNTYCODE_I", "000" },
+                { "CITYCODE_I", "00000" },
+                { "REPLACE", "0" },
+                { "COD1A", "Cardiopulmonary arrest" },
+                { "INTERVAL1A", "4 Hours" },
+                { "COD1B", "Eclampsia" },
+                { "INTERVAL1B", "3 Months" },
+                { "OTHERCONDITION", "hypertensive heart disease" },
+                { "DBPLACECITY", "Roanoke" },
+                { "SPOUSESUFFIX", "Ss" },
+                { "FATHERSUFFIX", "Sr" },
+                { "MOTHERSSUFFIX", "Ms" },
+                { "INFORMRELATE", "Friend of family" },
+                { "DISPSTATECD", "VA" },
+                { "DISPCITY", "Danville" },
+                { "FUNFACNAME", "Lancaster Funeral Home and Crematory" },
+                { "FUNFACADDRESS", "211 High Street" },
+                { "FUNCITYTEXT", "Lancaster" },
+                { "FUNZIP", "17573" },
+                { "PPDATESIGNED", "11132020" },
+                { "PPTIME", "2139" },
+                { "CERTFIRST", "Jim" },
+                { "CERTLAST", "Black" },
+                { "CERTSUFFIX", "Cr" },
+                { "CERTADDRESS", "44 South Street" },
+                { "CERTCITYTEXT", "Bird in Hand" },
+                { "CERTSTATECD", "PA" },
+                { "CERTSTATE", "Pennsylvania" },
+                { "CERTZIP", "17505" },
+                { "CERTDATE", "11142020" },
+                { "DTHCOUNTRYCD", "US" },
+                { "DTHCOUNTRY", "United States" },
+                { "PLACE1_1", "H" },
+                { "PLACE1_2", "I" },
+                { "PLACE8_1", "Hi 8_1" },
+                { "PLACE20", "Hi 20_1"}
+            };
+            // For each field, create a record, set that field, set all the other fields, and make sure the first field still has the same value
```

Comment:
I remember you said that setting the first name will clear a middle name, is middle name left out of this test?

Reply:
The comment says:
// MNAME, DMIDDLE, DDADMID, DMOMMID, SPOUSEMIDNAME, CERTMIDDLE (middle names behave oddly due to how FHIR represents names)

So, yes, these are left out, since we already know they are poorly behaved.   Pete added tests that are expected to fail for these.